src/components/Attendance.js

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect, useState } from "react"
+
+const Attendance = ({absences}) => {
+  let [statusColor, setStatusColor] = useState([ "bg-red-600", "bg-red-600", "bg-red-600", "bg-orange-500", "bg-orange-500", "bg-orange-500", "bg-yellow-300", "bg-yellow-300", "bg-yellow-300", "bg-green-500", "bg-green-500", "bg-green-500", "bg-green-600", "bg-green-600", "bg-green-600" ])
+
+  const absenceUpdate = (abs) => {
```

Comment:
Can you call your parameter `absences` since that is how it is referenced everywhere else in the component?